plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/contentassist/CustomN4JSParser.java

```diff
@@ -109,26 +108,6 @@ protected CustomInternalN4JSParser createParser() {
 		return result;
 	}
 
-	@Override
-	public void initializeFor(AbstractRule rule) {
```

Comment:
please consult @szarnekow about removing this

Reply:
IIRC the tests were red without this nasty patch. So it's safe to be removed when everythings green. Can be verified on master with a quick local build without this method.

Reply:
@szarnekow Thanks a lot for your super-fast review!